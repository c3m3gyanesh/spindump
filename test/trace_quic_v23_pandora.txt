Picoquic client against the Pandora server. Three attempts: regular connection, with retry, and then with HTTP/3 (which I think should fail, as Pandora does not support that yet). The commands that were used are:

