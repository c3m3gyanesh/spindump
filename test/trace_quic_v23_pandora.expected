[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "08b6b84e9f43f43c-ab6a12018e37fa1c (56214:4433)", "Ts": 1571179684955089, "State": "Starting", "Notes": "No rsp,v23,no spin", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "08b6b84e9f43f43c-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685132635, "State": "Up", "Notes": "No rsp,v23,no spin", "Right_rtt": 177546, "Packets1": 1, "Packets2": 0, "Bytes1": 1280, "Bytes2": 0, "Bandwidth1": 1280, "Bandwidth2": 0 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685133016, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 1, "Packets2": 4, "Bytes1": 1280, "Bytes2": 3453, "Bandwidth1": 1280, "Bandwidth2": 3453 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685133988, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 2, "Packets2": 6, "Bytes1": 1504, "Bytes2": 3769, "Bandwidth1": 1504, "Bandwidth2": 3769 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685133989, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 3, "Packets2": 6, "Bytes1": 1587, "Bytes2": 3769, "Bandwidth1": 1587, "Bandwidth2": 3769 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685328982, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 6, "Packets2": 8, "Bytes1": 3243, "Bytes2": 5684, "Bandwidth1": 3243, "Bandwidth2": 5684 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685531367, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 7, "Packets2": 10, "Bytes1": 3376, "Bytes2": 7219, "Bandwidth1": 3376, "Bandwidth2": 7219 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685531581, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 7, "Packets2": 11, "Bytes1": 3376, "Bytes2": 7446, "Bandwidth1": 3376, "Bandwidth2": 7446 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685734512, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 9, "Packets2": 11, "Bytes1": 3631, "Bytes2": 7446, "Bandwidth1": 3631, "Bandwidth2": 7446 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685734750, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 9, "Packets2": 12, "Bytes1": 3631, "Bytes2": 7673, "Bandwidth1": 3631, "Bandwidth2": 7673 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685872885, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 10, "Packets2": 13, "Bytes1": 3714, "Bytes2": 7761, "Bandwidth1": 3714, "Bandwidth2": 7761 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685872887, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 10, "Packets2": 14, "Bytes1": 3714, "Bytes2": 7834, "Bandwidth1": 3714, "Bandwidth2": 7834 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179685873523, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 10, "Packets2": 15, "Bytes1": 3714, "Bytes2": 7916, "Bandwidth1": 3714, "Bandwidth2": 7916 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "b6b84e9f43f43cac-08df4e9406d0f37d (56214:4433)", "Ts": 1571179686040004, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 11, "Packets2": 15, "Bytes1": 3779, "Bytes2": 7916, "Bandwidth1": 3779, "Bandwidth2": 7916 },
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "083fea70646f0a90-3d9477377c042430 (61794:4434)", "Ts": 1571179698914439, "State": "Starting", "Notes": "No rsp,v23,no spin", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08fd18ab20611e58 (61794:4434)", "Ts": 1571179699085598, "State": "Starting", "Notes": "V23,no spin", "Left_rtt": 701, "Packets1": 1, "Packets2": 1, "Bytes1": 1280, "Bytes2": 124, "Bandwidth1": 1280, "Bandwidth2": 124 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699264005, "State": "Up", "Notes": "V23,no spin", "Right_rtt": 178407, "Packets1": 2, "Packets2": 1, "Bytes1": 2560, "Bytes2": 124, "Bandwidth1": 2560, "Bandwidth2": 124 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699265475, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 3, "Packets2": 5, "Bytes1": 2704, "Bytes2": 3589, "Bandwidth1": 2704, "Bandwidth2": 3589 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699266084, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 4, "Packets2": 7, "Bytes1": 2870, "Bytes2": 3905, "Bandwidth1": 2870, "Bandwidth2": 3905 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699434303, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 5, "Packets2": 8, "Bytes1": 4370, "Bytes2": 3993, "Bandwidth1": 4370, "Bandwidth2": 3993 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699434308, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 5, "Packets2": 9, "Bytes1": 4370, "Bytes2": 4066, "Bandwidth1": 4370, "Bandwidth2": 4066 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699434833, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 5, "Packets2": 10, "Bytes1": 4370, "Bytes2": 4148, "Bandwidth1": 4370, "Bandwidth2": 4148 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "3fea70646f0a9051-08ac68422364e28c (61794:4434)", "Ts": 1571179699602124, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 6, "Packets2": 10, "Bytes1": 4435, "Bytes2": 4148, "Bandwidth1": 4435, "Bandwidth2": 4148 },
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "081c4c966af6a173-f3206c19ad639b43 (62935:4433)", "Ts": 1571179714907965, "State": "Starting", "Notes": "No rsp,v23,no spin", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.114.251","131.159.24.113"], "Session": "081c4c966af6a173-0825efa41a21eb59 (62935:4433)", "Ts": 1571179715080789, "State": "Up", "Notes": "No rsp,v23,no spin", "Right_rtt": 172824, "Packets1": 1, "Packets2": 0, "Bytes1": 1280, "Bytes2": 0, "Bandwidth1": 1280, "Bandwidth2": 0 }
]
